ebos/ecloutputblackoilmodule.hh

```diff
@@ -130,6 +130,7 @@ class EclOutputBlackOilModule : public EclGenericOutputBlackoilModule<GetPropTyp
 
     enum { conti0EqIdx = Indices::conti0EqIdx };
     enum { numPhases = FluidSystem::numPhases };
+    enum { numComponents = FluidSystem::numComponents };
```

Comment:
This value is, as far as I can tell, not used anywhere in this file.  Moreover, we have the same value [in our base class](https://github.com/OPM/opm-simulators/blob/e9645ada241736af52ab5ab97a8a47a6867050e0/ebos/eclgenericoutputblackoilmodule.hh#L280).  Is there a reason you need to introduce it here?